clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/services/LaunchTemplateService.java

```diff
@@ -135,6 +138,7 @@ public LaunchTemplate createLaunchTemplate(
     request.withNetworkInterfaces(
         new LaunchTemplateInstanceNetworkInterfaceSpecificationRequest()
             .withAssociatePublicIpAddress(settings.getAssociatePublicIpAddress())
+            .withIpv6AddressCount(associateIPv6Address != null && associateIPv6Address ? 1 : 0)
```

Comment:
associateIPv6Address is defaulted above to false so I don't think this will ever be null

Reply:
Good catch. Compulsive null checker